src/unasync/__init__.py

```diff
@@ -63,22 +63,22 @@ def match(self, filepath):
 
         return False
 
-    def unasync_file(self, filepath):
```

Comment:
Hmm, I actually use this API at work because it's a part of the existing code generator. All others I agree are private.

Something I've been thinking of is unasync should have a simple CLI, that'd fix the issue of not wanting to use it for a build system.

Reply:
Interesting! Can you share more about the use case?

Yeah, I guess a cli would be nice. For hip, I plan to use hip from nox, so a Python interface is enough.

I marked Rule.unasync_file as public, please take another look.

Reply:
I also use it from the Python interface: https://github.com/elastic/elasticsearch-py/pull/1203/files#diff-11742648bf5996c880c7a08632da8eddR305

Maybe I can put together something for CLI, I can see it being useful for people using `tox` and `pre-commit`.

Reply:
Wow, awesome! Sure, a cli would be useful, just not to us right now